srcpkgs/libresprite/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'libresprite'
+pkgname=libresprite
+version=1.0
+revision=1
+archs="x86_64"
```

Comment:
Nope. Unmark, rename hostmakedepends to makedepends and let's then look how to fix musl, i686 and arm.